Makefile

```diff
@@ -90,7 +90,6 @@ package-rpm-stayrtr: prepare
 	--url "$(URL)" \
 	--architecture $(ARCH) \
 	--license "$(LICENSE) "\
-	--package $(DIST_DIR) \
```

Comment:
Question: You removed this for `-t rpm`, does it need to stay for `-t deb` above?

Reply:
This option seems to work for DEB correctly, but for RPM it generates enormous packages - see:

Makefile: https://github.com/jacekkow/stayrtr/blob/v0.6.1-4/Makefile#L93
Results: https://github.com/jacekkow/stayrtr/releases/tag/v0.6.1-4

stayrtr-0.6.1_4-1.aarch64.rpm 180 MB
stayrtr-0.6.1_4-1.x86_64.rpm 82 MB
stayrtr_0.6.1-4_amd64.deb 16.7 MB
stayrtr_0.6.1-4_arm64.deb 15.7 MB


Makefile: https://github.com/jacekkow/stayrtr/blob/v0.6.1-5/Makefile#L93
Results: https://github.com/jacekkow/stayrtr/releases/tag/v0.6.1-5

stayrtr-0.6.1_5-1.aarch64.rpm 9.56 MB
stayrtr-0.6.1_5-1.x86_64.rpm 10.2 MB
stayrtr_0.6.1-5_amd64.deb 16.7 MB
stayrtr_0.6.1-5_arm64.deb 15.7 MB


I admit I did not investigate it further.

Reply:
Thanks for checking! I just noticed the inconsistent option